usaspending_api/etl/management/load_base.py

```diff
@@ -85,14 +84,15 @@ def handle(self, *args, **options):
             logger.info('Updating model description fields...')
             update_model_description_fields()
 
-        logger.info('Updating awards to reflect their latest associated transaction info...')
-        update_awards(tuple(award_update_id_list))
-
-        logger.info('Updating contract-specific awards to reflect their latest transaction info...')
-        update_contract_awards(tuple(award_contract_update_id_list))
```

Comment:
This code updates the transaction contract's base_and_all_options_value.  I have never seen this value used...